src/webview/git-import/gitImportLoader.ts

```diff
@@ -237,6 +200,74 @@ export default class GitImportLoader {
     }
 }
 
+async function close(event: any) {
+    try {
+        const selctedFolder: vscode.Uri = vscode.Uri.from(event.folder);
+        if (childProcess && event.notification !== 'Component created') {
+            forceCancel = true;
+            treeKill(childProcess.pid, async () => {
+                await deleteDirectory(selctedFolder.fsPath);
+            });
+        } else if (event.notification === 'Component created') {
```

Comment:
```suggestion
        } else if (event.notification === 'Component is successfully created.') {
```

Reply:
Here notification not shown to the user. This is for internal purpose that the close event called after/during the component creation. https://github.com/redhat-developer/vscode-openshift-tools/blob/23cf2f124bd2b5c7ed216a44644954790d0d204b/src/webview/git-import/app/gitImport.tsx#L168